corehq/apps/accounting/tests/test_customer_invoicing.py

```diff
@@ -345,143 +305,113 @@ class TestUserLineItem(BaseCustomerInvoiceCase):
-        num_users_advanced = self.advanced_rate.monthly_limit + 1
-        generator.arbitrary_commcare_users_for_domain(self.domain2.name, num_users_advanced)
+        num_users_main_domain = self.user_rate.monthly_limit + 10
+        generator.arbitrary_commcare_users_for_domain(self.main_domain.name, num_users_main_domain)
+        num_users_non_main_domain1 = 1
+        generator.arbitrary_commcare_users_for_domain(self.non_main_domain1.name, num_users_non_main_domain1)
 
         # Cover the cost of 1 User
         CreditLine.add_credit(
-            amount=Decimal(2.0000),
+            amount=Decimal(1.0000),
             feature_type=FeatureType.USER,
             account=self.account,
         )
 
         calculate_users_in_all_domains(self.invoice_date)
         tasks.generate_invoices_based_on_date(self.invoice_date)
+
+        num_to_charge = num_users_main_domain + num_users_non_main_domain1 - self.user_rate.monthly_limit
+
         self.assertEqual(CustomerInvoice.objects.count(), 1)
         invoice = CustomerInvoice.objects.first()
-        self.assertEqual(invoice.balance, Decimal(1509.0000))
+        self.assertEqual(invoice.balance, Decimal(1200.0000) + num_to_charge - 1)
 
     def test_subscription_level_user_credits(self):
         # Add User usage
-        num_users = self.user_rate.monthly_limit + 10
-        generator.arbitrary_commcare_users_for_domain(self.domain.name, num_users)
-        num_users_advanced = self.advanced_rate.monthly_limit + 1
-        generator.arbitrary_commcare_users_for_domain(self.domain2.name, num_users_advanced)
+        num_users_main_domain = self.user_rate.monthly_limit + 10
+        generator.arbitrary_commcare_users_for_domain(self.main_domain.name, num_users_main_domain)
+        num_users_non_main_domain1 = 1
+        generator.arbitrary_commcare_users_for_domain(self.non_main_domain1.name, num_users_non_main_domain1)
 
-        # Cover the cost of 1 User on the Standard subscription
+        # Cover the cost of 2 User
         CreditLine.add_credit(
             amount=Decimal(2.0000),
             feature_type=FeatureType.USER,
-            subscription=self.subscription
-        )
-        # Cover the cost of 5 Users on the Advanced subscription
-        CreditLine.add_credit(
-            amount=Decimal(10.0000),
-            feature_type=FeatureType.USER,
-            subscription=self.sub2
+            subscription=self.main_subscription
         )
 
+        num_to_charge = num_users_main_domain + num_users_non_main_domain1 - self.user_rate.monthly_limit
+
         calculate_users_in_all_domains(self.invoice_date)
         tasks.generate_invoices_based_on_date(self.invoice_date)
         self.assertEqual(CustomerInvoice.objects.count(), 1)
         invoice = CustomerInvoice.objects.first()
-        self.assertEqual(invoice.balance, Decimal(1500.0000))
-
-    def test_one_subscription_level_user_credit(self):
```

Comment:
Why remove this one?

Reply:
I think this one is redundant because we have `test_subscription_level_user_credits`. This two test did the same thing: adding subscription level credit.

Reply:
Hi Daniel, I think about the test again, and already add them back in 26f8dd921c48c4db7ef34eb14e0a00fa15e1692c and 641d475137a8ac1497cc06d1be4715acae45e459... After add them back, they did catch a bug in our customer invoicing code. I have no capacity to fix the bugs right now. I will ask Biyeun's advice.

Reply:
Hi Daniel, after discussing with Biyeun, I remove the two tests for multiple subscription level credits in de7570b82d3edc09306be807cc5b0817fc913b3e, because we have a bug in our code. Because multiple subscription level credits is not used, (though it can be used), so I think it is low priority. Kishan already created another ticket for that bug. And for now, I'll just comment out the tests.